modules/ROOT/pages/billing/how-to-pay.adoc

```diff
@@ -0,0 +1,159 @@
+= How to pay
+:description: Neo4j offers cloud services through two billing models.
+Pay-as-you-go, where usage is invoiced and charged on your credit card monthly in arrears.
+Prepaid, where credits are deducted from the available balance each month. 
+
+Aura usage is metered and aggregated based on billing dimensions, the result is priced according to Aura pricing plans.
+
+== Pay as you go
+
+* Get started by creating an Aura account. No sales team needed.
```

Comment:
I don't think this should be a bullet list. Better to write it as running text with complete sentences.